### ISSUE TITLE
Issues with `wit status` and `wit update`

### ISSUE THREAD

**Issue by jackkoenig** [sifive/wit#5]
This is not high priority, just wanted to record issues as I come across them

Here's a bash script that exercises problems with both `status` and `update`

```bash
#!/usr/bin/env bash

module load sifive/wit

DIR=$PWD
                                                                                                                                                                                
mkdir foo
cd foo
git init
touch foofile
git add foofile
git commit -m "Commit 1"
HASH1=`git rev-parse HEAD`

cd $DIR
wit init bar
cd bar
wit add $DIR/foo
git init
touch barfile
git add wit-manifest.json barfile
git commit -m "Commit 1 for bar"

cd $DIR/foo
touch anotherfile
git add anotherfile
git commit -m "Commit 2"
HASH2=`git rev-parse HEAD`

cd $DIR/bar
sed -i "s/$HASH1/$HASH2/g" wit-manifest.json

wit status
wit update
```
**TLDR** If you have a wit dependency on a repo and then that repo's remote gets updated, wit won't fetch from it automatically when you do `wit update`

I believe the issue with `status` is orthogonal but also comes up in this example so might as well mention both!

**Comment by jackkoenig** [comment]
I dunno what the issue with `status` was, but the lack of fetching is fixed.
